api/questions/views.py

```diff
@@ -70,3 +70,45 @@ def get(self, request, id=None):
         return Response({
             "error": "There are no questions"
         }, status=status.HTTP_404_NOT_FOUND)
+
+class QuestionEditViews(APIView):
+    """
+    class to edit question posted
+    """
+    permission_classes = [permissions.IsAuthenticated]
+    def put(self, request, id=None, m_id=None):
+        response = {}
+        try:
+            request.data['created_by'] = request.user
+            is_authenticated = request.user
+            owner = Question.objects.filter(created_by_id = is_authenticated).first()
+            meetup_id = Question.objects.filter(meetup_id=m_id).first()
+
+            if owner:
+
+                obj = Question.objects.get(id=id)
+                print(obj)
+                serializer = QuestionsSerializer(obj, data=request.data)
+                u_title = Question.objects.filter(
+                        title=request.data.get('title'))
+                u_body = Question.objects.filter(
+                        body=request.data.get('body'))
+                if serializer.is_valid():
+                    if u_title and u_body:
+                        return Response({
+                            "message": "Question is upto date"
+                    }, status=status.HTTP_400_BAD_REQUEST)
+                    Question.objects.update(
+                        title=request.data.get('title'),
+                        body=request.data.get('body')
+                    )
+                    return Response({"message":"question updated succesfully"}, status=status.HTTP_201_CREATED)
+                return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Hello @a-braham ,
This is a great work. I was wondering whether you could the number of return statements per method or function to one. It would make your methods and functions have a single entry point and a single exit point.